sklearn/ensemble/tests/test_bagging.py

```diff
@@ -309,7 +309,7 @@ def test_oob_score_classification():
                                                         iris.target,
                                                         random_state=rng)
 
-    for base_estimator in [DecisionTreeClassifier(), SVC()]:
+    for base_estimator in [DecisionTreeClassifier(), SVC(gamma="scale")]:
```

Comment:
I think we can use this PR to also set a `random_state`, to reduce diff of #8563.

It will anyway have a merge conflict for all these lines in that PR...

@jnothman Ok with you?